notebooks/vertex_genai/solutions/gemini_function_calling.ipynb

```diff
@@ -0,0 +1,1369 @@
+{
```

Comment:
<p>Same for this query.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/493/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>